src/test/scala/io/iohk/ethereum/vm/StaticCallOpcodeSpec.scala

```diff
@@ -0,0 +1,94 @@
+package io.iohk.ethereum.vm
+
+import io.iohk.ethereum.vm.Fixtures.blockchainConfig
+import io.iohk.ethereum.vm.MockWorldState._
+import org.scalatest.prop.PropertyChecks
+import org.scalatest.{Matchers, WordSpec}
+
+// scalastyle:off object.name
+class StaticCallOpcodeSpec extends WordSpec with Matchers with PropertyChecks {
+
+  val config = EvmConfig.ByzantiumConfigBuilder(blockchainConfig)
+  val startState = MockWorldState(touchedAccounts = Set.empty, noEmptyAccountsCond = true)
+
+  val fxt = new CallOpFixture(config, startState)
+
+  def stateWithProgram(code: Assembly): MockWorldState =
+    fxt.worldWithoutExtAccount.saveAccount(fxt.extAddr, fxt.accountWithCode(code.code)).saveCode(fxt.extAddr, code.code)
+
+  "STATICCALL" should {
+
+    "calling a program that executes a state-changing opcodes" should {
```

Comment:
I think testing of `Log` programs should be moved to other test, as they do not modify `staticcall.world` directly. They just append logs in program state, and world is modify after whole transactions.
The assertion in this test would be probably smoething like `staticcall.stateOut.logs.size shouldEqual 0` and 
`call.stateOut.logs.size shoud be > 0`